assignments/week-1/heart.k

```diff
@@ -0,0 +1,55 @@
+function getBeepers() {
+	while (beepersPresent()) {
+		pickBeeper();
+	}
+
+}
+
+function putBeepers() {
+	while (beepersInBag()) {
+		putBeeper();
+		move();
```

Comment:
nit: fn name should indicate that beepers are being put down horizontally giving communicating that two actions are occurring: putting and moving